Sources/Datadog/FeaturesIntegration/WebView/WKUserContentController+Datadog.swift

```diff
@@ -4,7 +4,7 @@
  * Copyright 2019-2020 Datadog, Inc.
  */
 
-#if DD_SDK_ENABLE_EXPERIMENTAL_APIS
+#if !os(tvOS) && DD_SDK_ENABLE_EXPERIMENTAL_APIS
```

Comment:
This basically means "no WebView tracking" / "sub apps" for tvOS. Do I see it right in the docs that [`WebKit` is unavailable on tvOS](https://developer.apple.com/documentation/webkit)?

Reply:
Yes, WebKit is not supported on tvOS so hybrid apps are not possible on this platform.